### DISCUSSION TITLE
Pemisahan data editing dan published

### DISCUSSION THREAD

**Discussion by igrir** [gimid/warga#8] [Ideas]
Saat ini data postingan yang dipublikasikan dan yang sedang ditulis adalah satu collection yang sama. Jadi ketika postingan sudah dipublish kemudian diedit, postingan tersebut langsung terupdate. Kepikiran aja, mungkin lebih baik dipisahkan antara penulisan dan publikasi? Jadi ketika postingan yang sudah dipublish kemudian ingin diedit kembali, ketika mencet "save" tidak langsung terupdate, tapi ketika mencet republish baru data tersebut terreplace

**Comment by reopucino** [comment]
semacam fitur wordpress juga, tapi karena nulisnya di Markdown saya hampir ga pernah pake feature ini. Kelebihanya lebih klo nulis langsung mungkin enak, bisa di cek2 dulu. Tanpa merusak tulisan lama.

Kekuranganya database bengkak, dan ada beberapa variasi dalam 1 tulisan / ada lebih dari 1 collection dari tulisan yang sama.

**Reply by igrir** [reply]
iya betul, databasenya bisa bengkak. Kayaknya bakal dibikin jadi dua collections saja om, yang published dan editing. Jadi cuma punya 1 variasi, si editing.

Menarik juga ternyata karena markdown fitur ini jadi nggak begitu penting ya?

**Reply by reopucino** [reply]
saya ga tau ini berlaku untuk semua orang apa ngga. Dulu waktu nulis di platform wordpress, biasanya saat saya nulis di notepad dulu. Terus tinggal copas, dan tinggal publish preview. Permasalahanya adalah terkadang tampilanya ga sesuai dengan keinginan (Ada yang salah atau ga cocok)

Nah dengan Markdown karena lebih simple dan markdown preview itu tampilanya ga jauh beda dengan platform yang berikan makanya saya jarang banget edit2, karena udah di edit manual dari sisi saya, tinggal copas aja.

==============

Back to topic, klo cuma 2 collections harusnya lebih okeh seh.. Dan jika collection yang sudah di publish collection lama bisa di apus.
